test/project_test.rb

```diff
@@ -32,12 +22,12 @@ def test_find_raises_if_not_found
 
   def test_schema
     project = Wordsmith::Project.find 'test'
-    expected = {a: 'Number', b: 'Number', c: 'Number'}
+    expected = { a: 'Number', b: 'Number', c: 'Number' }
```

Comment:
All other hashes i've seen in this PR have no spaces, so as long as you stay consistent one way or the other in this repo. Personally I like no spaces b/c then you can differentiate between blocks and hashes at a glance.